src/home_robot/home_robot/motion/stretch.py

```diff
@@ -902,6 +902,17 @@ def delta_hab_to_position_command(self, cmd, pan, tilt, deltas) -> List:
         tilt = tilt + deltas[9]
         return positions, pan, tilt
 
+    def config_to_manip_command(self, q):
+        """convert from general representation into arm manip config"""
```

Comment:
Slightly modifying the comment here to match the function name
```suggestion
        """convert from general representation into arm manip command"""
```

Reply:
done, thanks